### PR TITLE
Multiple persistence units follow up

### PR DESCRIPTION
**Pull request by gsmet**
@Sanne these are some small changes we talked about.

The Hibernate Validator is related to yesterday's PR: this way we won't miss bootstrap error anymore. Didn't include it in yesterday's PR because it would have conflicted with the Multi persistence units one.

### CODE REVIEW

**Review by Sanne — APPROVED** [review]
Sorry I forgot to explicitly marking this as approved. @gsmet not sure if you were planning to make any more changes so I'll leave it to you to press the merge button..

**Comment by gsmet** [comment]
@Sanne I added a commit along the lines of what we discussed for the storage engines. The MySQL/MariaDB test is a bit naive but I think it will do. We can refine later if someone complains.

File: extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsInconsistentStorageEnginesTest.java
Comment (Sanne) [inline extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsInconsistentStorageEnginesTest.java:38]: funny :)

**Review by Sanne — APPROVED** [review]

**Comment by Sanne** [comment]
thanks @gsmet ! It's not bullet-proof but I always merge progress ;)

**Comment by gsmet** [comment]
Yeah I agree, it's not perfect but better than nothing :).

File: extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsInconsistentStorageEnginesTest.java
Comment (gsmet) [reply extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsInconsistentStorageEnginesTest.java:38]: Yeah, I didn't want to end up with MySQL there :).

**Pull request merged**
